src/collection/reducer/index.ts

```diff
@@ -30,6 +30,7 @@ export function reducer(options: ReducerSchema): Rule {
     const reducerSpecSourceFile = readIntoSourceFile(host, stateDir.reducerSpec);
     const selectorsSourceFile = readIntoSourceFile(host, stateDir.selectors);
     const selectorsSpecSourceFile = readIntoSourceFile(host, stateDir.selectorsSpec);
+    const facadeSourceFile = readIntoSourceFile(host, stateDir.facade);
```

Comment:
If we pass `stateDir` to the `updateFacade` it's could be done within `updateFacade` method.